### PR TITLE
Provide formulas for the material-statistics postprocessor.

### PR DESCRIPTION
**Pull request by bangerth**
Better be specific with formulas than rely on a reader's understanding of
a text description.

I'm not sure there is a reason why one would ever want to use anything
other than the usual arithmetic averaging for the density and mass,
given that these have physical interpretations, so I dropped that remark.

I'm tacking on a change in wording in @gassmoeller's original changelog
to make it clear that the postprocessor computes 'the average density',
(passive) rather than 'averages the density' (an action).

Follow-up to #2630.

### CODE REVIEW

File: source/postprocess/material_statistics.cc [hunk 1]
```diff
@@ -120,8 +120,12 @@ namespace aspect
                                   "material statistics",
                                   "A postprocessor that computes some statistics about "
                                   "the material properties. In particular, it computes the "
-                                  "volume-averaged density, volume-averaged viscosity, and "
-                                  "total mass in the model. Currently, only arithmetic "
-                                  "averaging is supported.")
+                                  "volume-averages of the density and viscosity, and the "
+                                  "total mass in the model. Specifically, it implements "
+                                  "the following formulas in each time step: "
+                                  "$\\left<\\rho\\right> = \\frac{1}{|\\Omega|} \\int_\\Omega \\rho(\\mathbf x) \\, \\text{d}x, "
```
Comment (tjhei) [inline source/postprocess/material_statistics.cc]: I think you are missing a lot of $ signs in those lines.

**Comment by bangerth** [comment]
I fixed the formulas. I guess I am just so used that in emacs latex mode, when you type `$` it automatically creates the matching closing `$` as well :-)

**Comment by gassmoeller** [comment]
Hm, I am ok dropping that remark. @naliboff asked for it in the original pull request, because it might make more sense to compute a different average for the viscosity later on (e.g. averaging the logarithm of the viscosity). Anyway, we can always add it later on if we actually use different averaging schemes for different properties.

**Comment by bangerth** [comment]
ping?

**Pull request merged**